.github/scripts/review_agent.py

```
@@ -1,27 +1,59 @@
+        client = OpenAI(base_url=base_url, api_key=api_key)
+        retries = 0
+        while retries < 3:
+            try:
+                res = client.chat.completions.create(model=model, messages=[{"role": "user", "content": prompt}])
+                text = res.choices[0].message.content
+                break
+            except RateLimitError:
+                time.sleep(5 * (2 ** retries))
+                retries += 1
+                continue
+            except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)